style.css

```diff
@@ -282,10 +283,11 @@ ul.card-item {
   display: grid;
 }
 
-.pic3 {
+.pic4 {
   text-align: end;
-  margin: -3%;
-  position: relative;
+  margin-left: 85%;
+
+  /* position: relative; */
```

Comment:
- [ ] To make your code clean...
Please, clear your project from unused commented codes!

Reply:
done